modules/torch_compile.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p>Could we try these graphs with logarithmic y axes instead? These have large empty spaces that are less effective in communicating information.</p><p> </p><p>Are these plots from the run in this notebook? I don't see the code for generating them. If you've run the other notebook mentioned and generated these there, I would instead plot the times captured in this notebook and mention that times versus the fast_training_tutorial.ipynb times are better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1607/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
I have added the code for generating them and changed them to the plot captured in this notebook. And I am also using logarithmic y axes instead.